### PR TITLE
FILTER_BY options

### PR DESCRIPTION
**Pull request by sazzad16**

### CODE REVIEW

File: src/main/java/com/redislabs/redistimeseries/MultiRangeParams.java [hunk 1]
```diff
@@ -7,6 +7,9 @@
 
 public class MultiRangeParams {
 
+  private long[] timestamps;
```
Comment (gkorland) [inline src/main/java/com/redislabs/redistimeseries/MultiRangeParams.java:10]: @sazzad16 please rename these fields to something with filter. e.g `timestampsFilter`

File: src/main/java/com/redislabs/redistimeseries/MultiRangeParams.java [hunk 2]
```diff
@@ -7,6 +7,9 @@
...
+  private double[] values;
```
Comment (gkorland) [inline src/main/java/com/redislabs/redistimeseries/MultiRangeParams.java:11]:
```suggestion
    private double minFilter;
    private double maxFilter;
```

File: src/main/java/com/redislabs/redistimeseries/RangeParams.java [hunk 3]
```diff
@@ -7,6 +7,9 @@
 
 public class RangeParams {
 
+  private long[] timestamps;
```
Comment (gkorland) [inline src/main/java/com/redislabs/redistimeseries/RangeParams.java:10]: same as the comments on MultiRangeParams

**Review by gkorland — CHANGES_REQUESTED** [review]

File: src/main/java/com/redislabs/redistimeseries/MultiRangeParams.java (hunk 2 above)
Comment (sazzad16) [reply src/main/java/com/redislabs/redistimeseries/MultiRangeParams.java:11]: @gkorland With `double[] values`, we can do `if (values != null)`. With `double` we can't do such thing.

**Review by gkorland — APPROVED** [review]

**Review by gkorland — APPROVED** [review]

**Pull request merged**